tests/console/snapper_create.pm

```diff
@@ -21,7 +21,7 @@ sub get_last_snap_number {
     # that we got what we expect. See poo#25716
 
     # get snapshot id column
-    my $snap_head         = script_output("snapper list | head -n1");
+    my $snap_head         = script_output("snapper list | tail -n +1 | head -n1");
```

Comment:
Could you please put a comment explaining why we do this hack? Other than that LGTM.

Reply:
added